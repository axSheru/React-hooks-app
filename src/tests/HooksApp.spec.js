import { shallow } from 'enzyme';
import { HooksApp } from "../HooksApp";

describe('Pruebas en el HooksApp component.', () => {

    test('debe de hacer match con el snapshot.', () => {

        const wrapper = shallow( <HooksApp /> );

        expect( wrapper ).toMatchSnapshot();

    });

});