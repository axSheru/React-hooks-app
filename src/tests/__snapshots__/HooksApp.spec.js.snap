// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in HooksApp component. debe de hacer match con el snapshot. 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
